resolwe/storage/connectors/hasher.py

```diff
@@ -1,9 +1,37 @@
 """Compute hashes from stream."""
 import hashlib
 from io import RawIOBase
+from pathlib import Path
+from typing import TYPE_CHECKING, Dict
 
 import crcmod
 
+if TYPE_CHECKING:
+    from os import PathLike
+
+
+def compute_hashes(file_: "PathLike[str]") -> Dict[str, str]:
```

Comment:
`file` is not a reserved word in Python 3, so you don't need to add underscore.

Reply:
Nice to know. Will fix, code is much nicer without underscore.